src-plugins/manualRegistration/manualRegistration.h

```diff
@@ -19,6 +19,11 @@
 
 class manualRegistrationPrivate;
 
+namespace TransformName
+{
+    enum Value {RIGID,AFFINE};
+}
```

Comment:
Defining an enum of type Value in a namespace TransformName is a bit misleading. The enum itself should be called TransformName.